src/api/layer/balance_sheet.ts

```diff
@@ -1,21 +1,23 @@
-// import { BalanceSheet } from '../../types'
-// import { get } from './authenticated_http'
 import { BalanceSheet } from '../../types'
-import Data from './balance_sheet.json'
+import { get } from './authenticated_http'
 
-// export const getBalanceSheet = get<{
-//   data?: BalanceSheet
-//   error?: unknown
-// }>(
-//   ({ businessId }) =>
-//     `https://sandbox.layerfi.com/v1/businesses/${businessId}/balances`,
-// )
+export const getBalanceSheet = get<
+  GetBalanceSheetReturn,
+  GetBalanceSheetParams
+>(
+  ({ businessId, effective_date }: GetBalanceSheetParams) =>
```

Comment:
Mixed camelCase with snake_case:
```suggestion
  ({ businessId, effectiveDate }: GetBalanceSheetParams) =>
```